src/client/js/Utils/SvgManager.js

```diff
@@ -24,7 +24,7 @@ define([
     function isSvg(text) {
         var result = false;
 
-        if (DecoratorSVGIconList.indexOf(text) > -1) {
+        if (DecoratorSVGIconList.indexOf(text) > -1 || WebGMEGlobal.gmeConfig.client.allowUserDefinedSVG !== true) {
```

Comment:
by returning false here
the result of any svg lookup will end up being null that will indicate to use the default